src/lib/updateSiteUrlToLocal.ts

```diff
@@ -0,0 +1,38 @@
+import { SiteServer } from 'src/site-server';
+
+export const updateSiteUrlToLocal = async ( siteId: string ) => {
+	const server = SiteServer.get( siteId );
+	if ( ! server ) {
+		throw new Error( 'Site not found.' );
+	}
+
+	const { stdout: currentSiteUrl } = await server.executeWpCliCommand( `option get siteurl`, {
+		skipPluginsAndThemes: true,
+	} );
+
+	if ( ! currentSiteUrl ) {
+		console.error( 'Failed to fetch site URL after import' );
+		return;
+	}
+
+	const studioUrl = `http://localhost:${ server.details.port }`;
+	const oldUrl = currentSiteUrl.trim();
+	const urlWithoutProtocol = oldUrl.replace( /^https?:\/\//, '' );
+
+	const oldUrlVariants = [ `http://${ urlWithoutProtocol }`, `https://${ urlWithoutProtocol }` ];
+
+	for ( const urlToReplace of oldUrlVariants ) {
+		const { stderr, exitCode } = await server.executeWpCliCommand(
+			`search-replace '${ urlToReplace }' '${ studioUrl }' --skip-columns=guid`,
```

Comment:
The whole function is totally the same as it was before, so not necessary to review it, but two things:
1. Renamed to `updateSiteUrlToLocal`.
2. I added `--skip-columns=guid` to skip the `guid` column of posts, which is considered, AFAIU, as a best practice in WP world. I know that in this `Testing instructions` and similar cases it won't be a problem, but I believe that it's worth adding, since `guid` is just id, despite that fact that they look like urls.